templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,40 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List<a style="float: right" href="{% url 'taxi:manufacturer-create' %}">+</a></h1>
```

Comment:
Code Style issue: It's better to avoid inline styles. Consider moving this style to a CSS file.